version.bzl

```diff
@@ -18,7 +18,7 @@
 # This version should be updated together with the version of the Bazel
 # in .bazelversion.
 # TODO(alexeagle): assert this is the case in a test
-BAZEL_VERSION = "4.0.0"
+BAZEL_VERSION = "4.1.0"
```

Comment:
This is needed to be able to use `@platforms//:incompatible` to disable the  `diff_test` in `examples/pip_install`. I'm happy to revert this if there's a way to get it working, but I couldn't figure it out :(